pkg/bootstrapper/bootstrapper.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/pkg/errors"
 	"github.com/vincent-petithory/dataurl"
 	"golang.org/x/sys/windows/svc/mgr"
+
```

Comment:
Unnecessary new line. This will prevent the formatter from organizing this correctly.

Reply:
This is compliant with the regular formatting we do for imports

first group built in, second external imports, third imports from other packages within the module.

Reply:
Ah yes. Sorry my bad. Please ignore.